control-plane-operator/controllers/hostedcontrolplane/v2/assets/cluster-baremetal-operator/configmap-images.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-baremetal-operator-images
+  namespace: HCP_NAMESPACE
+  annotations:
+    capability.openshift.io/name: baremetal
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+data:
+  images.json: >
```

Comment:
These values don't look right, is something going to update them?

Reply:
Something probably should, noted